crowbar_framework/config/routes.rb

```diff
@@ -240,4 +236,48 @@
   # TODO(must): Get rid of this wildcard route
   match "/:controller/:action/*(:.format)",
     via: [:get, :post, :put, :patch, :delete]
+
+  namespace :api,
```

Comment:
Is it really necessary to put the API in a separate namespace? Won't the format and/or `Accept` header be enough to distinguish API requests from UI requests? What does the rest of the Rails community typically do here? AFAIK they don't tend to use `/api/` but ICBW.